src/lib/evil/unposix/meson.build

```diff
@@ -0,0 +1,41 @@
+project('evil_unposix', 'c')
+
+if target_machine.system() == 'windows'
+  subdir('sys')
+
+  evil_unposix_src = ['fnmatch.h',
+		      'getopt.h',
+		      'libgen.h',
+		      'limits.h',
+		      'locale.h',
+		      'math.h',
+		      'pthread.h',
+		      'semaphore.h',
+		      'stdlib.h',
+		      'string.h',
+		      'strings.h',
+		      'unimplemented.h',
+		      'unistd.h',
+	             ]
+
+  evil_unposix_lib = library('evil_unposix', evil_unposix_src,
+    include_directories: [include_directories('.')],
+    dependencies: [],
+    #install: true,
+    #version: meson.project_version(),
+  )
+
+  evil_unposix = declare_dependency(
+    include_directories: [include_directories('.')],
+    dependencies: [],
+    link_with: evil_unposix_lib,
+  )
+
+else
+  evil_unposix = declare_dependency()
+
+endif
+
+automatic_pkgfile = false
```

Comment:
Hm, I don't know how does this work. Could you explain? :)

(if you just copy'n'pasted from another meson.build, we can check this together)

Reply:
I noticed that its value is `true` only for the root meson.build, and then I just followed the pattern. It may has something to do with the installation.. I don't know.

Reply:
Oh okay, checked it. It is to generate a .pkg file and it is only true when it is not Windows.

Reply:
another (but compatible) variable is used here to set the value of `automatic_pkgfile` because `sys_windows` wasn't declared.